### PR TITLE
Default Community Files

### PR DESCRIPTION
**Pull request by aepfli**
Adding Default Community Files as a common default for all our projects.

### CODE REVIEW

File: .github/in-solidarity.yml
Comment (arminru) [inline .github/in-solidarity.yml]: empty line intentional?

File: .github/workflows/makefile.yml
Comment (arminru) [inline .github/workflows/makefile.yml]:
```suggestion
  # Triggers the workflow on push or pull request events but only for the default branch (usually main)
```

Comment (arminru) [reply .github/workflows/makefile.yml]: Is this even true? I think one would need to specify `pull_request: main` to narrow it down.

File: .github/workflows/makefile.yml
Comment (arminru) [inline .github/workflows/makefile.yml]: Keeping the tutorial comments reads a bit strange.

File: CODE_OF_CONDUCT.md
Comment (arminru) [inline CODE_OF_CONDUCT.md:1]: Is this just a copy of our existing one or did you adapt anything?

File: CONTRIBUTING.md
Comment (arminru) [inline CONTRIBUTING.md:11]: I would not give that away in here but refer to the exact legal terms in the CLA itself. Also, if the CLA would change we would need to think about updating this statement.

File: CONTRIBUTING.md
Comment (arminru) [inline CONTRIBUTING.md:14]:
```suggestion
(even if it was for a different project within this organization), you probably don't need to do it
```

File: CONTRIBUTING.md
Comment (arminru) [inline CONTRIBUTING.md:15]: This paragraph could also be omitted I think.

File: README.md
Comment (arminru) [inline README.md:32]: Is this for configuration within this repo or will it be inherited by all other repos?

File: README.md
Comment (arminru) [inline README.md]: Should this file be found in this PR here?

File: SECURITY.md
Comment (arminru) [inline SECURITY.md:5]:
```suggestion
Please report vulnerabilities to <opensource@dynatrace.com>.
If you would like to use encryption for your message, please first reach out to request a PGP key.
```
Maybe?

File: makefile
Comment (arminru) [inline makefile]: empty line intentional?

File: makefile
Comment (arminru) [inline makefile]: Is this a trustworthy image?

Comment (arminru) [inline makefile]: You might want to pin the version or otherwise the CI might suddenly fail in case of breaking changes which would require you to adapt to them before any other PRs can be merged.

File: CODE_OF_CONDUCT.md
Comment (aepfli) [reply CODE_OF_CONDUCT.md:1]: The default code of conduct, just in the newest version

File: README.md
Comment (aepfli) [reply README.md:32]: If proper usage of configuration was used during the development of the bot, then the app/bot supports also default configuration within the meta-project. But this is something which should work, depending on the bot. see https://probot.github.io/docs/best-practices/#store-configuration-in-the-repository

also an example: https://github.com/renovatebot/.github/tree/main/.github

File: makefile
Comment (aepfli) [reply makefile]: yes it is, i know it does not look like it but in fact igorshubovych is the maintainer of markdownlint-cli - https://github.com/igorshubovych/markdownlint-cli

File: .github/workflows/makefile.yml
Comment (aepfli) [reply .github/workflows/makefile.yml]: I removed all of the template comments - as they are not really adding any benefit.

Thank you

File: .github/in-solidarity.yml
Comment (aepfli) [reply .github/in-solidarity.yml]: removed, thank you

File: makefile
Comment (aepfli) [reply makefile]: thank you, I extracted the image into a variable and pointed it to the most current version (and the only version) :)

File: CONTRIBUTING.md
Comment (aepfli) [reply CONTRIBUTING.md:11]: I simply copied the Contributing guidelines from the current template. I will take your feedback and discuss this with @DavidPHirsch, if we want to tackle this now, or if we will generate a later task for improvements to this guideline file.

File: README.md
Comment (aepfli) [reply README.md]: my bad, I excluded this for now, as we want to take special care of this file on its own. within https://github.com/dynatrace-innovationlab we are using already a proper file, and this was actually the basis for this repository. - As this will be something we will provide within the near future, I will leave these lines for now.

File: CONTRIBUTING.md
Comment (aepfli) [reply CONTRIBUTING.md:11]: also it seems like we have been using the contribution guidelines from googles repo: https://github.com/google/new-project/blob/master/docs/contributing.md

File: .github/workflows/makefile.yml
Comment (dyladan) [inline .github/workflows/makefile.yml]: The name of this file `makefile.yml` does not describe its purpose

File: .github/workflows/makefile.yml
Comment (dyladan) [inline .github/workflows/makefile.yml]: This is really only markdown linting. Projects will still need to define their own linters

**Review by dyladan — COMMENTED** [review]
I could be wrong but I think any project that defines its own workflow will not get the workflow from this repo even if it is not the same one. Other than that all this generally looks good to me

File: .github/workflows/makefile.yml
Comment (aepfli) [reply .github/workflows/makefile.yml]: Actually, this file is only used within this repository and not shared. This is linting the readmes within this projects etc. To ensure a certain quality. I know it is really confusing at first glance, but workflow files are not shared

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (arminru) [inline .github/ISSUE_TEMPLATE/bug_report.md:2]: If a repo would like to use their own template, they would be able to override it with a file named `bug_report.md` at the same location, right?

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (arminru) [inline .github/ISSUE_TEMPLATE/bug_report.md]: `bug`, maybe

**Review by arminru — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (aepfli) [reply .github/ISSUE_TEMPLATE/bug_report.md:2]: yes exactly

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (aepfli) [reply .github/ISSUE_TEMPLATE/bug_report.md]: thank you, added and also `enhancement` to feature request

File: .github/workflows/makefile.yml
Comment (aepfli) [reply .github/workflows/makefile.yml]: thank you renamed the files

File: CONTRIBUTING.md
Comment (aepfli) [reply CONTRIBUTING.md:11]: we agreed to tackle this at a later stage, for now, we stick with the "google" default - and will improve this later with OSPO-37

**Pull request merged**
